test/parallel/test-node-run.js

```diff
@@ -0,0 +1,99 @@
+// Flags: --expose-internals
+'use strict';
+
+const common = require('../common');
+const { it, describe } = require('node:test');
+const assert = require('node:assert');
+
+const fixtures = require('../common/fixtures');
+const envSuffix = common.isWindows ? '-windows' : '';
+
+describe('node run [command]', () => {
+  it('should emit experimental warning', async () => {
+    const child = await common.spawnPromisified(
+      process.execPath,
+      [ 'run', 'test'],
+      { cwd: __dirname },
+    );
+    assert.match(child.stderr, /ExperimentalWarning: Task runner is an experimental feature and might change at any time/);
+    assert.match(child.stdout, /Can't read package\.json/);
+    assert.strictEqual(child.code, 1);
+  });
+
+  it('returns error on non-existent command', async () => {
```

Comment:
isn't this erroring because it doesn't find a package.json?
```suggestion
  it('returns error on non-existent file', async () => {
```

Reply:
That's correct. Nice catch. I'll update the test.